src/cavendish_particle_tracks/analysis.py

```diff
@@ -101,31 +101,39 @@ def _vars_to_show(self, calibrated=False):
 
     def _vars_to_save(self):
         """Variable to save in the output file, all for the moment"""
-        return self.__dict__.keys()
+        vars_to_save = [var for var in self.__dict__ if var[0] != "_"]
+        vars_to_save += ["rpoints", "dpoints", "spoints"]
+        return vars_to_save
 
     @property
     def rpoints(self):
```

Comment:
What are `rpoints` `dpoints` and `spoints`?
Radius... distance and stereoshift? How bad is it if you use full words? We have extra line length now...

Reply:
OK rather than bog you down here... let's make a separate issue to tackle once this is in.

- #176